README.md

```diff
@@ -1,2 +1,60 @@
 # hw01
 Repository for homework 01
```

Comment:
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     x      |              |            |
| **Coding strategy**         |      x     |              |            |
| **Presentation**            |    x       |              |            |
| **Achievement, creativity** |     x      |              |            |
| **Ease of access**          |    x       |              |            |

Remarks:

* This was a nicely formatted, clean and readable biography.
* I especially liked the placement and choice of photos.
* You definitely went above and beyond demonstrating different markdown styles!
* If I had to criticize, I would say that you should delete the 'image:' lines and let the images speak for themselves.

Really nice work!!